src/Compilers/CSharp/Test/Syntax/Diagnostics/DiagnosticTest.cs

```diff
@@ -419,6 +424,11 @@ public void NullableWarnings()
                     ErrorCode.WRN_AnalyzerReferencesFramework,
                     ErrorCode.WRN_UnreadRecordParameter,
                     ErrorCode.WRN_DoNotCompareFunctionPointers,
+                    ErrorCode.WRN_CallerArgumentExpressionParamForUnconsumedLocation,
```

Comment:
>ErrorCode.WRN_CallerArgumentExpressionParamForUnconsumedLocation, [](start = 20, length = 65)

It looks like this test requires unnecessary maintenance. Perhap the check for the range of error codes at the beginning of the loop should be adjusted instead? Based on the comment only "Nullable-unrelated warnings in the C# 8 range should be added to this array."

Reply:
@AlekseyTs I guess you meant this to be a "general" feedback on this test, rather than this PR specifically. Is that right?
Should I target `main` for that change in a separate PR, or file an issue for discussion?

Reply:
> I guess you meant this to be a "general" feedback on this test, rather than this PR specifically. Is that right?

It is specific to this PR because there are changes here.

> Should I target main for that change in a separate PR, or file an issue for discussion?

I guess either way is fine as long as we either fix it in this PR, or make sure it will be fixed before merging the feature into main.

---
In reply to: [607929347](https://github.com/dotnet/roslyn/pull/51952#discussion_r607929347) [](ancestors = 607929347)

Reply:
Sorry I initially misunderstood your comment. Fixed in 418ec88444bc21ba3ca815f3a78f29a8b8743b75.